src/rs.cpp

```diff
@@ -462,6 +462,13 @@ const char * rs2_aus_get_counter_data( const rs2_strings_list * buffer, int i, r
 }
 HANDLE_EXCEPTIONS_AND_RETURN( 0, buffer )
 
+const rs2_raw_data_buffer * rs2_aus_receive_json_data(rs2_error ** error) BEGIN_API_CALL
+{
+    std::vector<uint8_t> buffer_aus_data_recieved = librealsense::aus_get_data();
+    return new rs2_raw_data_buffer { buffer_aus_data_recieved };
```

Comment:
I don't remember, where do we delete this "new" instance?

Reply:
There is a func in rs.cpp already implemented:
void rs2_delete_raw_data(const rs2_raw_data_buffer* buffer);
If the user calls this func he  will need to delete the rs2_raw_data_buffer* buffer he gets at the end.

Reply:
Then we need to state it in the API comment IMO
Like here:
https://github.com/IntelRealSense/librealsense/blob/b874e42685aed1269bc57a2fe5bf14946deb6ede/include/librealsense2/h/rs_device.h#L106

Reply:
Added